package/Dockerfile

```diff
@@ -15,12 +15,12 @@ WORKDIR /var/lib/rancher
 
 ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
-ARG SYSTEM_CHART_DEFAULT_BRANCH=release-v2.6
-ARG CHART_DEFAULT_BRANCH=release-v2.6
+ARG SYSTEM_CHART_DEFAULT_BRANCH=dev-v2.6
+ARG CHART_DEFAULT_BRANCH=dev-v2.6
 ARG PARTNER_CHART_DEFAULT_BRANCH=main
 ARG RKE2_CHART_DEFAULT_BRANCH=main
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go
-ARG CATTLE_KDM_BRANCH=release-v2.6
+ARG CATTLE_KDM_BRANCH=dev-v2.6
```

Comment:
There is one more reference here we should update https://github.com/rancher/rancher/blob/release/v2.6/pkg/settings/setting.go#L59

Reply:
Also here https://github.com/rancher/rancher/blob/release/v2.6/Dockerfile.dapper#L15

Reply:
seems to be addressed